addons/mail/wizard/mail_blacklist_remove.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models
+
+
+class MailBlacklistRemove(models.TransientModel):
+    _name = 'mail.blacklist.remove'
+    _description = 'Remove email from blacklist wizard'
+
+    @api.model
+    def _default_email(self):
+        return self.env.context.get('email')
```

Comment:
You could simply set default_email in context and everything will be managed by standard behavior (default_get check for default_field_name in context).

Reply:
Neat, I didn't know that existed. Updated it!